buildSrc/src/main/groovy/com/thoughtworks/go/build/docker/Distro.groovy

```diff
@@ -54,7 +54,7 @@ enum Distro implements DistroBehavior {
       return [
         '# install glibc and zlib for adoptopenjdk',
         '# See https://github.com/AdoptOpenJDK/openjdk-docker/blob/ce8b120411b131e283106ab89ea5921ebb1d1759/8/jdk/alpine/Dockerfile.hotspot.releases.slim#L24-L54',
-        '  apk add --no-cache --virtual .build-deps curl binutils',
```

Comment:
I see we use curl later here. ls it not needed?

Reply:
We have already added curl